### PR TITLE
Github Actions -- Fix composite

### PR DESCRIPTION
**Pull request by oseasmoran73**
## Description

Slack notification have silently been failing due to composite actions. This PR allows secret to be passed in (which was root cause) rather than composite verify secret.

## Testing done

Latest Run

### CODE REVIEW

File: script/github-actions/check-broken-links.js [hunk 1]
```diff
@@ -21,16 +21,13 @@ if (fs.existsSync(reportPath)) {
   const color = shouldFail ? '#D33834' : '#FFCC00'; // danger or warning, needs to be in hex
   const summary = brokenLinks.summary;
   const heading = `${brokenLinks.brokenLinksCount} broken links found in ${envName} \\n\\n <${SERVER_URL}>`; // TODO: Add @cms-helpdesk when prod ready
-  const slackBlocks = `[{"type": "section","text": {"type": "mrkdwn","text": "${heading}"}}]`;
```
Comment (oseasmoran73) [inline script/github-actions/check-broken-links.js:24]: `Blocks` section only used for broken links. Can't default empty string. Removing and appending to `attachment`

File: .github/workflows/continuous-integration.yml [hunk 2]
```diff
@@ -261,7 +261,6 @@ jobs:
           slack_app_token: ${{ env.SLACK_APP_TOKEN }}
           slack_bot_token: ${{ env.SLACK_BOT_TOKEN }}
           attachments: ${{ steps.get-broken-link-info.outputs.SLACK_ATTACHMENTS }}
-          blocks: ${{ steps.get-broken-link-info.outputs.SLACK_BLOCKS }}
```
Comment (oseasmoran73) [inline .github/workflows/continuous-integration.yml:264]: `Blocks` section only used for broken links. Can't default empty string. Removing and appending to `attachment`

File: .github/workflows/slack-notify/action.yml [hunk 3]
```diff
@@ -8,20 +8,22 @@ inputs:
   attachments:
     description: attached message
     required: true
-  blocks:
```
Comment (oseasmoran73) [inline .github/workflows/slack-notify/action.yml:11]: `Blocks` section only used for broken links. Can't default empty string. Removing and appending to `attachment`

File: .github/workflows/continuous-integration.yml [hunk 4]
```diff
@@ -439,45 +438,6 @@ jobs:
           path: cypress/results
           retention-days: 1
 
-      - name: Configure AWS credentials
```
Comment (oseasmoran73) [inline .github/workflows/continuous-integration.yml:442]: Removing section as it is part of slack-notify workflow

**Review by k80bowman — APPROVED** [review]

**Pull request merged**